lib/spack/spack/util/web.py

```diff
@@ -50,29 +47,8 @@ class HTMLParseError(Exception):
 # Timeout in seconds for web requests
 _timeout = 10
 
-# See docstring for standardize_header_names()
-_separators = ('', ' ', '_', '-')
-HTTP_HEADER_NAME_ALIASES = {
-    "Accept-ranges": set(
-        ''.join((A, 'ccept', sep, R, 'anges'))
-        for A, sep, R in product('Aa', _separators, 'Rr')),
-
-    "Content-length": set(
-        ''.join((C, 'ontent', sep, L, 'ength'))
-        for C, sep, L in product('Cc', _separators, 'Ll')),
-
-    "Content-type": set(
-        ''.join((C, 'ontent', sep, T, 'ype'))
-        for C, sep, T in product('Cc', _separators, 'Tt')),
-
-    "Date": set(('Date', 'date')),
-
-    "Last-modified": set(
-        ''.join((L, 'ast', sep, M, 'odified'))
-        for L, sep, M in product('Ll', _separators, 'Mm')),
-
-    "Server": set(('Server', 'server'))
-}
+# See docstring for get_header()
+_UNFUZZ_HEADER = lambda header: re.sub(r'[ _-]', '', header).lower()
```

Comment:
This isn't a constant; it's a function.  Just `def` it.  Better yet, `def` it as `unfuzz()` inside the only function where it is used.